src/Utilities/validationUtilities.js

```diff
@@ -1,4 +1,5 @@
 const validateEmail = (email, errorMessage) => {
+    //   eslint-disable-next-line
```

Comment:
no need for this comment either since specific to environment even though we should be using eslint. Feel free to make an issue regarding eslint integration and then a pr going over how to use it.